synth.py

```diff
@@ -21,17 +21,21 @@
 logging.basicConfig(level=logging.DEBUG)
 
 # run the gapic generator
-gapic = gcp.GAPICGenerator()
-versions = ['v1beta1']
+gapic = gcp.GAPICMicrogenerator()
+versions = ['v1beta1','v1beta2']
```

Comment:
Do we need to ship both `v1beta1` and `v1beta2`?  Could we just ship 2 and keep it simple?

Reply:
Removed v1beta1. It sounds like it won't ever become public, so we can remove it.